pkg/controllers/provisioningv2/provisioningcluster/template.go

```diff
@@ -384,9 +384,9 @@ func machineDeployments(cluster *rancherv1.Cluster, capiCluster *capi.Cluster, d
 				Template: capi.MachineTemplateSpec{
 					ObjectMeta: capi.ObjectMeta{
 						Labels: map[string]string{
-							capi.ClusterLabelName:           capiCluster.Name,
+							capi.ClusterNameLabel:           capiCluster.Name,
```

Comment:
Duplicate

Reply:
Hi, capr.ClusterNameLabel and capi.ClusterNameLabel are different. I don't think its duplicate.